### PR TITLE
Remove unused deps after TS convert

### PR DESCRIPTION
**Pull request by jaketrent**
- prop-types
- filter-prop-types

### CODE REVIEW

**Review by danethurber — APPROVED** [review]
looks good. better merge it before i merge the ActionMenu changes ...

**Comment by jaketrent** [comment]
^ For real?  Is there an order you'd prefer for actionmenu-something?

**Comment by danethurber** [comment]
no. you merge first. i'm happy to do the conflicts

**Comment by jaketrent** [comment]
k, I'll wait til the pipeline is green, then merge.

File: packages/actionmenu/src/react/item.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ import React, {
 } from 'react'
 
 import stylesheet from '../css'
-import { tagName } from '../vars'
+import { tagName as tagNames } from '../vars'
```
Comment (EdwardIrby) [inline packages/actionmenu/src/react/item.tsx:15]: Should we just rename this tagNames

**Review by EdwardIrby — APPROVED** [review]

File: packages/actionmenu/src/react/item.tsx (hunk 1 above)
Comment (jaketrent) [reply packages/actionmenu/src/react/item.tsx:15]: nbd, but imo, yes. I avoided just so that I didn't have to bump the major version. :)

**Pull request merged**